app/controllers/users_controller.rb

```diff
@@ -61,6 +61,14 @@ def destroy
     redirect_to users_url, notice: exception.message
   end
 
+  def orders
+    @order = User.find(session[:user_id]).orders
```

Comment:
create a method `current_user` in application controller, and use it throughout.
Also, whats the diff b/w `find` vs `find_by`?
What if user_id is nil?

Reply:
find is used only when searching by primary key and find_by requires an attribute.

Reply:
done and if the user id is nil it redirected to the login page.

Reply:
done